versions/3.0.md

```diff
@@ -1998,293 +1915,136 @@ paths:
     - name: id
       in: path
       required: true
-      description: the user identifier, as userId or username
+      description: the user identifier, as userId 
       schema:
         type: string
-    responses:
-      '200':
-        description: the user being returned
-        content:
-          application/json:
-            schema:
-              type: object
-              properties:
-                uuid: the unique user id
-                  type: string
-                  format: uuid
-```
-
-Can be used in a link like this:
-
-```yaml
-Addresses:
-  # the target link operationId
-  operationId: getUserAddress
-  parameters:
-    # get the `id` field from the request path parameter named `id`
-    userId: '{$request.path.id}'
+    get:
+      responses:
+        '200':
+          description: the user being returned
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  uuid: the unique user id
+                    type: string
+                    format: uuid
+        links:
+          address:
+            # the target link operationId
+            operationId: getUserAddress
+            parameters:
+              # get the `id` field from the request path parameter named `id`
+              userId: $request.path.id
+  # the path item of the linked operation
+  /users/{userid}/address:
+    parameters:
+    - name: userid
+      in: path
+      required: true
+      description: the user identifier, as userId 
+      schema:
+        type: string
+      # linked operation
+      get:
+        operationId: getUserAddress
+        responses:
+          '200':
+            description: the user's address
 ```
 
-Where the `$request.path.id` is the value passed in the request to `/users/{id}`.
-
-##### Response Payload Example
+When a runtime expression evaluates to null, no parameter value is passed to the target operation.
```

Comment:
I see the need to allow falling back to the default and think this should be specified here (I assume you are referring to the `default` value define in a JSON schema for an operation parameter - please confirm.)  

However, there is a difference between the value at the JSON pointer being null (valid JSON) compared to the item the JSON path refers to not existing. If the JSON pointer value is null, I think that should be passed; it may be significant (see my other comment about null).